docs/developer-center/new-guides/quickstart.md

```diff
@@ -1,185 +1,966 @@
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+We can see the variable that contains the population for 2019 is the one with the slug `POPCY_f5800f44`. Finally, we are ready to enrich our areas of influence with that variable.
+
+
+```python
+from cartoframes.data.observatory import Variable
+from cartoframes.data.observatory import Enrichment
+
+variable = Variable.get('POPCY_f5800f44')
+
+isochrones_df = Enrichment().enrich_polygons(isochrones_df, [variable])
+isochrones_df.head()
 ```
 
-![Bikeshare data](../img/new-guides/quickstart/analyze_helper.png)
 
-Awesome! You have finished with your analysis and see that your company has done a good job placing the stations.
 
-### Publish and share your results
 
-To finish your work, you want to share the results with some teammates. Also, it would be great if you could allow them to play with the information. Let's do that!
+<div>
+<style scoped>
+    .dataframe tbody tr th:only-of-type {
+        vertical-align: middle;
+    }
+
+    .dataframe tbody tr th {
+        vertical-align: top;
+    }
+
+    .dataframe thead th {
+        text-align: right;
+    }
+</style>
+<table border="1" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th></th>
+      <th>data_range</th>
+      <th>geometry</th>
+      <th>POPCY</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>0</th>
+      <td>900</td>
+      <td>MULTIPOLYGON (((-73.95934 40.68011, -73.96074 ...</td>
+      <td>1221.455789</td>
+    </tr>
+    <tr>
+      <th>1</th>
+      <td>900</td>
+      <td>MULTIPOLYGON (((-73.96187 40.58632, -73.96289 ...</td>
+      <td>1534.248133</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>900</td>
+      <td>MULTIPOLYGON (((-73.99082 40.62694, -73.99170 ...</td>
+      <td>1311.667005</td>
+    </tr>
+    <tr>
+      <th>3</th>
+      <td>900</td>
+      <td>MULTIPOLYGON (((-74.02851 40.64062, -74.02882 ...</td>
+      <td>1179.999175</td>
+    </tr>
+    <tr>
+      <th>4</th>
+      <td>900</td>
+      <td>MULTIPOLYGON (((-74.00110 40.60186, -74.00249 ...</td>
+      <td>1334.454790</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+Great! Let's see the result in a map:
+
+
+```python
+from cartoframes.viz.helpers import color_continuous_layer
+
+Map(color_continuous_layer(isochrones_df, 'POPCY', 'Population'))
+```
+
+<div class="example-map">
+    <iframe
+        id="quickstart_guide_starbucks_brooklyn_pop"
+        src="https://cartoframes.carto.com/kuviz/2f8f2c92-f8ed-4d1d-8e09-04ef85ad80dd"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
 
-First, you have to upload the data used by your maps to CARTO using the [Dataset]() class:
 
-```py
-from cartoframes.data import Dataset
+We can see that the area of influence of the store on the right, it's the one with the largest population. Let's go a bit further and calculate and visualize the average revenue per person.
```

Comment:
replace: "We can see that the area of influence of the store on the right, it's the one with the largest population."
with: "We can see that the area of influence of the store on the right, is the one with the highest population."